testdata/1_input

```diff
@@ -0,0 +1,8 @@
+pick 4365d5d [DROP][INVESTIGATE] Weird shell diffs
```

Comment:
Hm, comments in the files will not work. Not as long as I have the current setup.